src/mscorlib/shared/System/Runtime/InteropServices/MemoryMarshal.cs

```diff
@@ -23,5 +24,32 @@ public static class MemoryMarshal
         /// </remarks>
         public static Memory<T> AsMemory<T>(ReadOnlyMemory<T> readOnlyMemory) =>
             Unsafe.As<ReadOnlyMemory<T>, Memory<T>>(ref readOnlyMemory);
+        
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ref T GetReference<T>(in Span<T> span) => ref span.Reference;
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ref readonly T GetReference<T>(in ReadOnlySpan<T> span) => ref span.Reference;
```

Comment:
I do not think the Reference property will get inlined here for the class case. We may want to make the field internal instead.

Reply:
The `ByReference<T> _pointer` field?

Reply:
Yes.

Reply:
Yes, it looks like without `AggressiveInlining` it doesn't get inlined.